rj_gameplay/stp/skill/__init__.py

```diff
@@ -21,10 +21,12 @@ class Skill(ABC):
     @abstractmethod
     def tick(self, world_state: stp.rc.WorldState) -> RobotIntent:
         """Logic for Skill goes here. RobotIntents obtained via Actions.
+        self.robot is updated forably and quietly.
         .msg
                 :param world_state: Current world state.
                 :return: A single RobotIntent.
         """
+        self.robot = world_state.our_robots[self.robot.id]
```

Comment:
When running BasicDef, I got an error claiming NoneType has no attr 'id', which suggests self.robot is None on a tick(). I believe the issue is actually that sometimes the init functions for certain skills take in None as their default param (e.g. skill/receive), but I haven't tested that hypothesis. Can you remove that default param and see if that fixes it?